### DISCUSSION TITLE
RE: Log4j

### DISCUSSION THREAD

**Discussion by jjhiza** [AsamK/signal-cli#832] [Q&A] [answered]
Just wanted to drop a question regarding Log4j Java library, and the new zero-day vulnerability that has been discovered. Does Signal itself, or this CLI implementation make use of any technologies that could be impacted by this vulnerability? Again, just wanted to ask the question publicly so users can rest assured that the CLI implementation remain safe and secure. Thank for all the hard work!

**Comment by AsamK** [comment, marked as answer]
Neither signal-cli, nor its dependencies use log4j.

**Reply by technillogue** [reply]
It's probably fine, but maybe worth checking if there are any potential vulnerabilities via http://slf4j.org/log4shell.html ?
